test/client/fixtures/sandbox/fetch-test.js

```diff
@@ -245,7 +248,10 @@ if (window.fetch) {
                     .then(function (headers) {
                         // NOTE: The Fetch API's Request.credentials property now defaults to "same-origin" per the latest
                         // revision of the specification. https://developer.mozilla.org/en-US/Firefox/Releases/61
-                        if (browserUtils.isFirefox && browserUtils.version > 60)
```

Comment:
We cannot need to check a browser native behavior. Therefore, let's change assertion to
```js
ok(headers.hasOwnProperty(xhrHeaders.fetchRequestCredentials));
```

Reply:
Fixed in https://github.com/DevExpress/testcafe-hammerhead/pull/1694